tests/test_config_parsing/test_enkf_obs_parsing.py

```diff
@@ -17,12 +18,12 @@
 @pytest.mark.filterwarnings("ignore::UserWarning")
 @pytest.mark.filterwarnings("ignore::RuntimeWarning")
 @pytest.mark.filterwarnings("ignore::ert.parsing.ConfigWarning")
-@pytest.mark.usefixtures("set_site_config")
```

Comment:
it seems like contrary to the PR text, here you're not using the fixture anymore?

Reply:
The fixture is not necessary when we skip parsing.

Reply:
the PR body should be modified then - it literally says

> Use set_siteconfig fixture to avoid parsing.

but you're not using the fixture anymore.